distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -957,6 +957,50 @@ public void testPostGetRecordByIdProductRetrievalWithInvalidRange()
         deleteMetacard(metacardId);
     }
 
+    @Test
+    public void testGetMetacardResourceStatusNotCached()
+            throws IOException, XPathExpressionException {
+        String fileName = testName.getMethodName() + ".txt";
+        String metacardId = ingestXmlWithProduct(fileName);
+
+        String productDirectory = new File(fileName).getAbsoluteFile()
+                .getParent();
+        urlResourceReaderConfigurator.setUrlResourceReaderRootDirs(new String[] {
+                DEFAULT_URL_RESOURCE_READER_ROOT_RESOURCE_DIRS, productDirectory});
+
+        String url = REST_PATH.getUrl() + "sources/ddf.distribution/" + metacardId;
+        String resourceCachedXpath = String.format("//*[@name='%s']/*/text()", Metacard.RESOURCE_CACHE_STATUS);
+
+        get(url).
+                then().
+                body(hasXPath(resourceCachedXpath, is("false")));
+
+        deleteMetacard(metacardId);
+    }
+
+    @Test
+    public void testGetMetacardResourceStatusCached()
+            throws IOException, XPathExpressionException {
+        String fileName = testName.getMethodName() + ".txt";
+        String metacardId = ingestXmlWithProduct(fileName);
+
+        String productDirectory = new File(fileName).getAbsoluteFile()
+                .getParent();
+        urlResourceReaderConfigurator.setUrlResourceReaderRootDirs(new String[] {
+                DEFAULT_URL_RESOURCE_READER_ROOT_RESOURCE_DIRS, productDirectory});
+
+        String url = REST_PATH.getUrl() + "sources/ddf.distribution/" + metacardId;
```

Comment:
I think that the `REST_PATH` field happens to have a trailing slash, while other fields like it do not. Changing `REST_PATH` to exclude the trailing `/` would mean changing any places that currently use it, so i just left it for now